tests/Unit/Entities/Notifications/test_NotificationEvent.py

```diff
@@ -0,0 +1,202 @@
+from json import loads
+from pytest import mark
+from importlib import import_module
+
+from paddle_billing.Entities.Notifications import NotificationEvent
+
+from paddle_billing.Notifications.Entities.Subscription import Subscription
+from paddle_billing.Notifications.Entities.SubscriptionCreated import SubscriptionCreated
+
+from tests.Utils.ReadsFixture import ReadsFixtures
+
+
```

Comment:
```suggestion
```

Reply:
This is a flake8 codestyle, removing this line results in error:
```
tests/Unit/Entities/Notifications/test_NotificationEvent.py:12:1: E302 expected 2 blank lines, found 1
```